### ISSUE TITLE
[FEATURE]: (Front end) Add placeholder text to tell user what file formats are acceptable

### ISSUE THREAD

**Issue by prrao87** [whyhow-ai/knowledge-table#13]
## What

There are two `+` symbols in the frontend UI. The left one (though not immediately obvious) allows users to upload a document.

![image](https://github.com/user-attachments/assets/44cc47b4-df37-4a54-92f3-afcd07b78cac)

1. Is it possible to make it more explicit at first glance how a user can add documents? I had to stare at the UI for a few seconds to figure out what to do (there's no placeholder text or text labels with any instructions)
2. Does it make sense to add some placeholder text within the header telling the user what document formats are accepted? (e.g., `.pdf`, `.txt`, `.json`, etc.)? 

In my case,  I tried uploading a `.txt` file by clicking the `+` on the left side, and nothing happened. I added all the required keys to the `.env` file in the backend and ran the front end and backend via `docker-compose.yml`, and nothing happens -- the UI remains unchanged.

## Why

It's not immediately obvious what to do once the UI is opened. Then, once the left `+` symbol is clicked, loading `.txt` file(s) does nothing (the UI remains unchanged, and it seems that there's no action taken because exporting triples results in an empty file being generated). This is despite setting the OpenAI + Unstructured API keys, and setting the Milvus DB name and password in `.env` and running the whole setup via `docker-compose.yml`.

```json
{
  "triples": [],
  "chunks": []
}
```

## Implementation guidance

Some placeholder text placed strategically to tell the user about a) what to do the moment they access the UI, and b) what document formats are acceptable would be helpful.

**Comment by prrao87** [comment]
Another thing that's not immediately obvious: What is the intent of the front end UI?

- Is it for users to manually add their entities and relationships? 
- Can users add more than one document, or is the `+` symbol only meant to add one document at a time?
- is an Unstuctured.io API key required to handle vanilla text files? Or is it required only to handle PDF and more complex documents?

So many questions! cc @chiajy2000

**Comment by tomsmoker** [comment]
Hey @prrao87, Unstructured is optional and PyPDF is the default behaviour regardless of document type.  We're using the same LangChain document loaders for both, and it can handle text files.  Can view more here:

https://github.com/whyhow-ai/knowledge-table/blob/main/backend/src/knowledge_table_api/services/document.py

Odd that it didn't work.  Can you send the text on here? Or otherwise on DM? I'll try and resolve.

**Comment by tomsmoker** [comment]
_Is it for users to manually add their entities and relationships, or is it for users to upload their documents so that they can be processed by the backend?_

It's for users to add documents, and then those documents can be turned into structured output through natural language questions.  The output format can be constrained by the search box, and specific columns can be referred to with the {...} syntax. 

_Can users add more than one document, or is the + symbol only meant to add one document at a time?_

You can add as many as you want.  I usually batch load them in.

**Comment by prrao87** [comment]
Hey @tomsmoker, I'd like to only work with text files for now, and here's the file I tried to upload.

I'm trying to model data from the following three text files as a graph (with vector chunks alongside):

[fink_1.txt](https://github.com/user-attachments/files/17328173/fink_1.txt)
[kapito_1.txt](https://github.com/user-attachments/files/17328174/kapito_1.txt)
[wagner_1.txt](https://github.com/user-attachments/files/17328175/wagner_1.txt)

Here's the schema I want to capture, expressed as `(:source_label) -[:relationship_label]-> (:target_label)`:

```py
schema = [
    ("PERSON", "STUDIED_AT", "UNIVERSITY"),
    ("PERSON", "IS_CEO_OF", "ORGANIZATION"),
    ("PERSON", "IS_FOUNDER_OF", "ORGANIZATION"),
    ("PERSON", "BORN_IN", "CITY"),
    ("CITY", "IS_CITY_IN", "STATE"),
]
```

I'd fully expect there to be unresolved entities that are duplicates of one another (e.g., "BlackRock" and "BlackRock Inc", etc.), so I'd love to see what features we could look at to address these sorts of discrepancies.

**Comment by tomsmoker** [comment]
This has been addressed in #27.
